components/ThemeToggleButton.jsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from "react";
+import styles from "@/styles/components/theme-toggle-button.module.scss";
+import PageFeatures from "@/server/context";
+import ToggleButton from "@/components/ToggleButton";
+
+export default function ThemeToggleButton() {
+	const theme = useContext(PageFeatures)["theme"];
+	const setTheme = useContext(PageFeatures)["setTheme"];
+	const ToggleTheme = () => {
```

Comment:
pedantic warning! pls put whitespace between above and below this function :')

Reply:
noted